client/blocks/app-banner/index.jsx

```diff
@@ -262,6 +262,13 @@ function BannerIcon( { icon } ) {
 			autoplay: true,
 			path: icon,
 		} );
+
+		const reducedMotion = window.matchMedia( '(prefers-reduced-motion: reduce)' ).matches;
+
+		if ( reducedMotion ) {
+			animation.goToAndStop( 145, true );
```

Comment:
My only concern is whether the animations would ever get longer than this 😁 

Maybe there's a way to do it dynamically?

Reply:
Interesting. It looks like we may need to await configuration ready. Also, I wonder if it makes sense to avoid autoplaying when the user prefers reduced motion. 

```diff
diff --git a/client/blocks/app-banner/index.jsx b/client/blocks/app-banner/index.jsx
index 52b4c522b4..3e165828da 100644
--- a/client/blocks/app-banner/index.jsx
+++ b/client/blocks/app-banner/index.jsx
@@ -255,18 +255,20 @@ export class AppBanner extends Component {
 
 function BannerIcon( { icon } ) {
 	useEffect( () => {
+		const reducedMotion = window.matchMedia( '(prefers-reduced-motion: reduce)' ).matches;
+
 		const animation = lottie.loadAnimation( {
 			container: document.querySelector( '.app-banner__icon' ),
 			renderer: 'svg',
 			loop: false,
-			autoplay: true,
+			autoplay: ! reducedMotion,
 			path: icon,
 		} );
 
-		const reducedMotion = window.matchMedia( '(prefers-reduced-motion: reduce)' ).matches;
-
 		if ( reducedMotion ) {
-			animation.goToAndStop( 145, true );
+			animation.addEventListener( 'config_ready', () => {
+				animation.goToAndPlay( animation.totalFrames, true );
+			} );
 		}
 
 		return () => animation.destroy();
```

Reply:
Thanks David! I've gone ahead to implement your suggestion in https://github.com/Automattic/wp-calypso/pull/67826/commits/f61ab8aaf6a361500e8872f88429684ef5d7e5b8.

> Also, I wonder if it makes sense to avoid autoplaying when the user prefers reduced motion.

I think so, it was necessary to have this enabled when using `goToAndStop`, but not necessary with `goToAndPlay`, so I've followed your suggestion. 🙇‍♀️